<launch>
    <rosparam>/use_sim_time: 0</rosparam>
    <rosparam file="$(find trajectory_planner)/cfg/planner_test.yaml" />

    <param name="odom_topic" value="/hummingbird/odometry_sensor1/odometry"/>
    <param name="goal_topic" value="/red/exploration/goal"/>
    <param name="path_topic" value="/planning/uav_path"/>
    <param name="pose_topic" value="/planning/uav_path_poses"/>

    <!-- <param name="odom_topic" value="/hummingbird/odometry_sensor1/odometry"/>
    <param name="goal_topic" value="/uav1/exploration/goal"/>
    <param name="path_topic" value="/planning/uav_path"/>
    <param name="pose_topic" value="/planning/uav_path_poses"/> -->
    <param name="map_path" value="/home/manh/pathplaning_ws/src/trajectory_planner/map/lv4_raw.binvox.bt"/>
    


    <node pkg="trajectory_planner" type="test3" name="test3" output="screen" />

    <arg name="map_location" default="$(find trajectory_planner)/map/lv4_raw.binvox.bt"/>
    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server_node" args="$(arg map_location)" output="log" />
</launch>
